website/client/components/chat/chatCard.vue

```diff
@@ -217,6 +216,21 @@ export default {
       return 'Message hidden (shadow-muted)';
     },
   },
+  mounted () {
+    const links = this.$refs.markdownContainer.getElementsByTagName('a');
+    for (let i = 0; i < links.length; i++) {
+      const link = links[i];
+      if (links[i].getAttribute('href').startsWith('/profile/')) {
```

Comment:
what's the default action that's prevented here? I mean what's wrong with keeping the normal link? that it opens them in a new window?

Reply:
Yes, exactly. Normally it would open in a new tab, but this way it opens in the same window as the modal. (same as when you click on their username at the top of a chat message.). This solution also still allows users to CMD+Click to open in a new tab in the background.